### ISSUE TITLE
[Feature] Adding Copy button on top of codes for improved user experience

### ISSUE THREAD

**Issue by NateNear** [colbyfayock/next-cloudinary#309]
# **Feature Request**

## **Is your feature request related to a problem? Please describe.**

 For applying any code or feature its hard to copy all the code and might even result in copying the wrong code or leaving a bracket or two while selecting all. 

## **Describe the solution you'd like**

It would be really helpful if a "Copy" button is present that copies everything in just a single click. It solve a munch of hassles

For example:

![image](https://github.com/colbyfayock/next-cloudinary/assets/92938055/d99d8ba0-3f05-4af9-9ffe-a402bad50993)

which is not present in the current website 
![image](https://github.com/colbyfayock/next-cloudinary/assets/92938055/f8bcdf32-e31a-4ccc-a0e0-72ad067a54f2)

**Comment by colbyfayock** [comment]
great point @NateNear - i think generally the code blocks could include a better experience including syntax highlighting

**Comment by NateNear** [comment]
can I work on this issue pls

**Comment by colbyfayock** [comment]
yes sure! before adding anything custom, i would explore how the Nextra docs is achieving both highlighting and the copy button as i'm wondering if there is a built-in way to handle this:

https://nextra.site/docs/guide/built-ins/callout#usage

that example uses `mdx` as the language:

https://github.com/shuding/nextra/blob/8ff0576d60d2fabb4bb9f0d64ee4261928195ba7/docs/pages/docs/guide/built-ins/callout.mdx#L34

the Next Cloudinary docs are also a couple versions behind, very possible thats a new feature too

**Comment by NateNear** [comment]
aight! I'll look into the docs and add a PR.

**Comment by NateNear** [comment]
@colbyfayock I've made a PR resolving this issue. 
Kindly look into it. :))
